iroh-net/src/lib.rs

```diff
@@ -243,14 +243,16 @@ mod magicsock;
 pub mod metrics;
 pub mod netcheck;
 pub mod ping;
-pub mod relay;
-pub mod stun;
+mod relay_map;
 pub mod ticket;
 pub mod tls;
+
 pub(crate) mod util;
 
 pub use endpoint::{AddrInfo, Endpoint, NodeAddr};
 pub use iroh_base::{key, key::NodeId};
+pub use iroh_relay as relay;
+pub use relay_map::{RelayMap, RelayMode, RelayNode, RelayUrl};
```

Comment:
Ah I see now.

Initially I thought "why do `mod relay_map` & `pub use relay_map::*` instead of just keeping a `pub mod relay`?", but now I see the `pub use iroh_relay as relay` re-export below.

:thinking: that said, I think it should be possible to write a `relay.rs` module that re-exports all of `iroh_relay` at its own level & also has `RelayMap`, `RelayMode`, `RelayNode` & `RelayUrl`.
That would be slightly less breakage for users. May or may not be worth it?